starters/features/storybook/.storybook/main.ts

```diff
@@ -0,0 +1,33 @@
+import type { StorybookViteConfig } from '@storybook/builder-vite';
+import { mergeConfig, UserConfig } from 'vite';
+import baseConfig from '../vite.config';
+
+const config: StorybookViteConfig = {
+  addons: ['@storybook/addon-essentials'],
+  stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
+  framework: '@storybook/html',
+  core: {
+    builder: '@storybook/builder-vite',
+  },
+  features: {
+    storyStoreV7: true,
+  },
+  viteFinal: async (defaultConfig) => {
```

Comment:
Would it be better to make a `qwik-vite` [storybook framework](https://storybook.js.org/docs/react/api/new-frameworks#gatsby-focus-wrapper) with this configuration instead of using @storybook/html and adding these configurations in every consuming project?